com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -652,7 +658,7 @@ private void AddInterpolatedState(NetworkTransformState newState)
 
         private void OnNetworkStateChanged(NetworkTransformState oldState, NetworkTransformState newState)
         {
-            if (!NetworkObject.IsSpawned)
+            if (!NetworkObject.IsSpawned || !gameObject.activeInHierarchy)
```

Comment:
Where does the `gameObject.activeInHierarchy` bit comes from ? I don't see how it ties in with the rest.

Reply:
Yeah, that needs to go away.  It was from an earlier iteration.